run.py

```diff
@@ -9,6 +13,32 @@ def run():
     video_info = youtube_dl.YoutubeDL().extract_info(
         url=video_url, download=False
     )
+
+    vedio_title = video_info['title']
+    filename = f"{vedio_title}.mp3"
+
+    # Ask the user for the path they want to save the file after it has been downloaded
+    path_to_save = input(
+        r"Enter where the file is to be saved or leave empty if you want it to be saved in the current directory: ")
+    if path_to_save == "":
+        where_to_save = os.getcwd()
+    else:
+        where_to_save = f"{path_to_save}"
```

Comment:
Can you declare `where_to_save` above line 21, this way you can do a `!=` check and can get rid of the `else` check